specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json

```diff
@@ -62,13 +62,17 @@
     },
     "/app/governance/members": {
       "get": {
-        "operationId": "GetConsortiumMembers",
+        "operationId": "ListConsortiumMembers",
         "x-ms-examples": {
-          "GetConsortiumMembers": {
-            "$ref": "./examples/ledger/GetConsortiumMembers.json"
+          "ListConsortiumMembers": {
+            "$ref": "./examples/ledger/ListConsortiumMembers.json"
           }
         },
-        "summary": "Gets the consortium members.",
+        "x-ms-pageable": {
+          "itemName": "consortium",
```

Comment:
Seems it's should be `members`:
https://github.com/Azure/azure-rest-api-specs/blob/e08e3258433d86cd0f8623c00cafb9a3b8979fdd/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json#L516